src/Compatibility/Microsoft.DotNet.PackageValidation/Microsoft.DotNet.PackageValidation.csproj

```diff
@@ -17,4 +17,8 @@
   <ItemGroup>
     <EmbeddedResource Update="Resources.resx" GenerateSource="true" />
   </ItemGroup>
+
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.DotNet.PackageValidation.Tests" Key="0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9" />
```

Comment:
We usually never do this. For testing internals, we almost always prefer compiling implementation files on the test project instead.

Reply:
Oh, I see that we were already doing this. You don't have to fix this now, but we should definitely get rid of the InternalsVisibleTo as this is just messy.

Reply:
I don't like InternalsVisibleTo either but that's unrelated to this change. If we feel strongly about it, let's open an issue to track the work.